### DISCUSSION TITLE
Can we use vscode in godot??

### DISCUSSION THREAD

**Discussion by leamandeep** [godotengine/godot-proposals#4984] [Editor]
Hey !! Can we use vscode as a editor in godot??

**Comment by theraot** [comment]
Yes.

*I'll answer here, but know Godot has a dedicated Q&A site: https://godotengine.org/qa/*

You can open the project folder on Visual Studio Code. And Godot will pick the changes to the files when it gains focus. *The same idea applies for using Visual Studio (not Code) for C# Godot projects, except in that case you want to open the project file that Godot generates.*

You might be interested in the Visual Studio Code extensions such as godot-tools and C# Tools for Godot. You might be interested in configuring the editor path in godot-tools settings to launch Godot from Visual Studio Code, which will also enable debugging. It is also possible to configure Visual Studio Code as external code editor in the Editor Settings in Godot. Which allows Godot to launch Visual Studio Code. *Refer to the readme of the mentioned extensions for more information.*

And if you really want to, you can disable the integrated editor via Manage Editor Feature Profiles in Godot (but why?).

**Reply by leamandeep** [reply]
Thanks for giving your precious time to my question🙏🙏
